python/mlcroissant/mlcroissant/_src/structure_graph/nodes/field.py

```diff
@@ -233,3 +233,12 @@ def parent(self) -> Node:
         if parent:
             return parent
         raise ValueError(f"field={self} does not have any parent.")
+
+    @property
+    def is_subfield(self) -> bool:
```

Comment:
You don't seem to use this method. Is this still needed?

Reply:
Removed for simplicity